frontend/packages/dev-console/integration-tests/views/serverless.view.ts

```diff
@@ -0,0 +1,42 @@
+import { browser, ExpectedConditions as until, by, element } from 'protractor';
+import * as sidenavView from '../../../../integration-tests/views/sidenav.view';
+import * as crudView from '../../../../integration-tests/views/crud.view';
+import { switchPerspective, Perspective, sideHeader } from './dev-perspective.view';
+
+export const firstKebabMenu = element.all(by.css('[data-test-id="kebab-button"]')).first();
+export const actionItems = element(by.css('data-test-id="action-items"'));
+export const editCount = element.all(by.css('.pf-c-dropdown__menu-item')).first();
+export const machineSetsInput = element(
+  by.className('pf-c-form-control co-m-number-spinner__input'),
+);
+export const submitCount = element(by.id('confirm-action'));
+export const cellClass = element(by.cssContainingText('td.col-sm-4.hidden-xs', '9 of 9 machines'));
+export const pageSidebar = element(by.id('page-sidebar'));
+export const installServerlessOperator = element(
+  by.css('div.pf-c-modal-box.co-catalog-page__overlay.co-catalog-page__overlay--right'),
+);
+export const createProject = element(by.id('yaml-create'));
+export const nameofProject = element(by.id('input-name'));
+export const displayNameofProject = element(by.id('input-display-name'));
+
+// Creating namespace from UI
+export const createNewProject = async (projectName: [string, string]) => {
+  await browser.wait(until.presenceOf(sidenavView.navSectionFor('Home')));
+  await sidenavView.clickNavLink(['Home', 'Projects']);
+  await crudView.isLoaded();
+  await browser.wait(until.presenceOf(createProject));
+  await createProject.click();
+  await browser.wait(until.presenceOf(nameofProject));
+  await nameofProject.sendKeys(projectName[0]);
+  await displayNameofProject.sendKeys(projectName[1]);
+  await submitCount.click();
+};
+
+// Refresh Page
+export const hardRefreshPage = async () => {
+  await switchPerspective(Perspective.Developer);
+  expect(sideHeader.getText()).toContain('Developer');
+  await switchPerspective(Perspective.Administrator);
+  expect(sideHeader.getText()).toContain('Administrator');
+  await browser.wait(until.visibilityOf(pageSidebar));
+};
```

Comment:
this looks more like util for `switchPerspective` for refresh can we use `browser.refresh();` ?

Reply:
I tried using the browser.refresh() also but it didn't help that's why I tried the switching the perspective.

Reply:
yeah all am saying is `hardRefreshPage` is misleading as you are not doing refresh there

Reply:
I will try to use the browser.refresh and will check again.